libs/providers/go-feature-flag-web/README.md

```diff
@@ -0,0 +1,83 @@
+
+// You can now use the client to use your flags
+if(client.getBooleanValue('my-new-feature', false)){
+    //...
+}
+
+// You can add handlers to know what happen in the provider
+client.addHandler(ProviderEvents.Ready, () => { ... });
+client.addHandler(ProviderEvents.Error, () => { //... });
+client.addHandler(ProviderEvents.Stale, () => { //... });
+client.addHandler(ProviderEvents.ConfigurationChanged, () => { //... });
+```
+
+### Available options
+| Option name                   | Type   | Default        | Description                                                                                                                                                                                     |
+|-------------------------------|--------|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| endpoint                      | string |                | endpoint is the URL where your GO Feature Flag server is located.                                                                                                                               |
+| apiTimeout                    | number | 0 = no timeout | (optional) timeout is the time in millisecond we wait for an answer from the server.                                                                                                            |
+| apiKey                        | string |                | (optional) If GO Feature Flag is configured to authenticate the requests, you should provide an API Key to the provider. Please ask the administrator of the relay proxy to provide an API Key. |
+| websocketRetryInitialDelay    | number | 100            | (optional) initial delay in millisecond to wait before retrying to connect the websocket                                                                                                        |
+| websocketRetryDelayMultiplier | number | 2              | (optional) multiplier of websocketRetryInitialDelay after each failure _(example: 1st connection retry will be after 100ms, second after 200ms, third after 400ms ...)_                         |
+| websocketMaxRetries           | number | 10             | (optional) maximum number of retries before considering the websocket unreachable                                                                                                               |
+
+### Reconnection
+If the connection to the GO Feature Flag instance fails, the provider will attempt to reconnect with an exponential back-off.   
+The `websocketMaxRetries` can be specified to customize reconnect behavior.
+
+### Event streaming
+The `FlagdWebProvider` receives events from flagd with changes. Combined with the event API in the web SDK, this allows for subscription to flag value changes in clients.
+
+```typescript
+client.addHandler(ProviderEvents.Ready, () => {
```

Comment:
You may want to switch this to a flag value changed example to match the text above, plus it's a more interesting feature.